src/parsers/power_models_data.jl

```diff
@@ -98,10 +98,23 @@ function _get_pm_dict_name(device_dict)
     return name
 end
 
+function _get_pm_bus_name(device_dict::Dict, unique_names::Bool)
+    if haskey(device_dict, "name")
+        if unique_names
+            name = strip(device_dict["name"])
+        else
+            name = strip(device_dict["name"]) * "_" * string(device_dict["bus_i"])
+        end
+    else
+        name = strip(join(string.(device_dict["source_id"]), "-"))
```

Comment:
Is it possible and practical to always use this name? I assume that it will be easier for everyone to understand if this parser always produces bus names in the same way.